packages/@okta/vuepress-site/docs/guides/submit-app/prepare-for-submission/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Prepare your app for submission
+---
+
+After you have built a functioning app integration in either [SAML](/docs/guides/build-sso-integration/) or [SCIM](/docs/guides/build-provisioning-integration/), a few steps are required to submit it for Okta review.
+
+Your submission must provide Okta with all the metadata needed to create a customized app for publication in the [Okta Integration Network](https://www.okta.com/integrations/).
+
+Okta analysts review the submission, create the customized app, run it through Okta internal quality assurance testing, and then publish it to the OIN.
+
+Complete the following steps before you submit your SCIM app:
+
+1. Check the [profile attributes and their mappings](/docs/guides/a-build-provisioning-integration/attribute-mapping/) for your app.
+1. [Test your Okta integration](/docs/guides/a-build-provisioning-integration/test-scim-app/) using the Okta SCIM CRUD test suite on Runscope.
+1. Prepare your [customer-facing configuration guide](/docs/guides/a-build-provisioning-integration/prepare-guide/).
+1. Optionally, [create a demo video](/docs/guides/a-build-provisioning-integration/prepare-guide/#create-a-demo-video) showing a working integration.
```

Comment:
I'd argue that the link doesn't really provide any additional information, and so isn't real worth linking to.

Reply:
Link removed.